io.openems.edge.controller.ess.timeofusetariff/src/io/openems/edge/controller/ess/timeofusetariff/ControlMode.java

```diff
@@ -0,0 +1,31 @@
+package io.openems.edge.controller.ess.timeofusetariff;
+
+import io.openems.common.types.OptionsEnum;
+
+public enum ControlMode implements OptionsEnum {
+	CHARGE_CONSUMPTION(0, "Charge consumption from from grid"), //
+	DELAY_DISCHARGE(1, "Dealys discharge for high price hours"); //
```

Comment:
"Delays"; how about "Delays discharge during low-price hours?"

Reply:
Makes sense.
Anyways it should have been "Delays discharge to be usable during high-price hours", your suggestion is easier to understand.